pallets/parachain-staking/src/lib.rs

```diff
@@ -1480,7 +1490,13 @@ pub mod pallet {
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+
 		fn on_initialize(n: T::BlockNumber) -> Weight {
+			// base weight of a "no-op" on_initialize() call
+			// TODO: rewrite passive_on_initialize() and active_on_initialize() benchmarks to work as
+			// described here
```

Comment:
Before, the `active_on_initialize` was the weight function during round transitions and `passive_on_initialize` was the weight function during blocks without the round transition.

Now that the payout is spread over multiple blocks, I recommend merging `passive_on_initialize` and `active_on_initialize` into one `on_initialize` benchmark. Note that `passive_on_initialize` was never used as a base weight and should not be used as a base weight now.

Reply:
should follow name conventions set [here](https://github.com/PureStake/moonbeam/pull/1035#discussion_r766820951)